geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -247,7 +207,7 @@ public Result createRegion(
               new Object[] {CliStrings.CREATE_REGION__USEATTRIBUTESFROM, useAttributesFrom}));
```

Comment:
Bit of a nit-pick, but we have been trying to clean up these cases where we unnecessarily create new object arrays to pass to varargs methods. This occurs several places in the class

Reply:
If there is one person to whom you need not apologize for nit-picking, it's this guy.

Inspections find six cases across `RegionCreateFunction` and `CreateRegionCommand`.  Done and done.